Makefile

```diff
@@ -4,27 +4,31 @@ SHELL=/bin/bash -euo pipefail -O globstar
 
 all:
 	make clean > build.log
+	make check-licenses >> build.log
 	make build >> build.log
 	make test >> build.log
 	make release >> build.log
 
-.PHONY: install build test publish release clean
+.PHONY: all
 
 install: install-validator install-node install-python install-hooks
 
 build: build-specification build-coordinator build-validator build-proxies
 
-test: validate-models check-licenses test-coordinator
+test: validate-models test-coordinator
 	cd tests/e2e/pact && make test
 	poetry run pytest ./scripts/update_prescriptions.py
 
 publish:
 	echo Publish
 
 release:
-	mkdir -p dist
+	mkdir -p dist/pact/models
 	cp -r specification/dist/. dist
-	rsync -av --progress --copy-links tests/e2e/pact dist --exclude node_modules
+	rsync -av --progress --copy-links tests/e2e/pact dist --exclude pact/node_modules --exclude pact/pact
+	rm -f dist/pact/tsconfig.json && mv dist/pact/tsconfig-deploy.json dist/pact/tsconfig.json
+	rsync -av --progress --copy-links models dist/pact --exclude models/library/node_modules --exclude models/build
+	rsync -av --progress --copy-links coordinator dist/pact --exclude coordinator/node_modules
```

Comment:
make pact asset for running smoke-tests during deploy, bundle up model library and coordinator projects